CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewBottomNavigationBar.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  ReviewBottomNavigationBar.swift
+//  
+//
+//  Created by David Vizaknai on 24.10.2022.
+//
+
+import UIKit
+
+protocol ReviewBottomNavigationBarDelegate: AnyObject {
+    func didTapMainButton(on navigationBar: ReviewBottomNavigationBar)
+    func didTapSecondaryButton(on navigationBar: ReviewBottomNavigationBar)
+}
+
+final class ReviewBottomNavigationBar: UIView {
+    private let configuration = GiniConfiguration.shared
+    lazy var mainButton: MultilineTitleButton = {
+        let button = MultilineTitleButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.layer.cornerRadius = 16
```

Comment:
[Suggestion] I think we should have a button.layer.cornerRadius as a separate customization option 😉